contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md

```diff
@@ -0,0 +1,370 @@
+---
+kep-number: draft-20172411
+title: My First KEP
+authors:
+  - "@chrislovecnm"
+owning-sig: sig-contributor-experience
+participating-sigs:
+  - sig-aws
+  - sig-cluster-lifecycle
+reviewers:
+  - TBD
+  - "@parispittman"
+approvers:
+  - TBD
+  - "@sarahnovotny"
+editor:
+  - TBD
+creation-date: 2017-22-10
+last-updated: 2017-24-11
+status: draft
+see-also:
+  - TBD
+replaces:
+  - "N/A"
+superseded-by:
+  - "N/A"
+---
+
+# Kubernetes Pilot Mentor Guide
+
+## Table of contents
+
+* [Kubernetes Pilot Mentor Guide](#kubernetes-pilot-mentor-guide)
+   * [What is this document about](#what-is-this-document-about)
+   * [Benefits of Being a Mentor](#benefits-of-being-a-mentor)
+   * [How is a Mentor Successful](#how-is-a-mentor-successful)
+      * [Mentors Growth](#mentors-growth)
+   * [Mentor Conduct](#mentor-conduct)
+      * [Communication](#communication)
+      * [Code of Conduct](#code-of-conduct)
+   * [Successful Interaction with Mentees](#successful-interaction-with-mentees)
+      * [Patience](#patience)
+      * [Communication](#communication-1)
+      * [Goal Setting](#goal-setting)
+      * [Building Confidence](#building-confidence)
+      * [Developing Trust](#developing-trust)
+   * [Learning Styles](#learning-styles)
+      * [Auditory](#auditory)
+      * [Kinesthetic](#kinesthetic)
+      * [Visual](#visual)
+      * [Credit](#credit)
+   * [Community Culture Element to Impart](#community-culture-element-to-impart)
+   * [Time Expectations](#time-expectations)
+   * [When to ask for help](#when-to-ask-for-help)
+   * [Summary](#summary)
+
+## What is this document about
+
+This document is a guide for Mentors involved in the Kubernetes Pilot Program.
```

Comment:
Link to more details on the pilot program? How do ppl sign up, when does it start, who is eligible, etc

Reply:
Should be in the readme in the directory of this folder, yes we need that info, but should we have that in this doc as well?

Reply:
Sorry if i jumped the gun, i was curious. Will wait :)

Agree that there is no need to repeat it here. Linking to another doc that has that info should be sufficient.

Reply:
No worries! This is a alpha for this program, we are just figuring out everything ourselves. Ping me on slack if you have questions.